gateway/api/proxymanager/proxymanager.go

```diff
@@ -43,13 +43,15 @@ func Get(c *gin.Context) {
 		return
 	}
 	c.JSON(http.StatusOK, &openapi.ProxyManagerResponse{
-		ID:                    obj.ID,
-		Status:                openapi.ClientStatusType(obj.Status),
-		RequestConnectionName: obj.RequestConnectionName,
-		RequestPort:           obj.RequestPort,
-		RequestAccessDuration: obj.RequestAccessDuration,
-		ClientMetadata:        obj.ClientMetadata,
-		ConnectedAt:           obj.ConnectedAt.Format(time.RFC3339),
+		ID:                       obj.ID,
+		Status:                   openapi.ClientStatusType(obj.Status),
+		RequestConnectionName:    obj.RequestConnectionName,
+		RequestConnectionType:    obj.RequestConnectionType,
+		RequestConnectionSubType: obj.RequestConnectionSubType,
```

Comment:
The proxy manager state table is a way of passing the state from the HTTP API to the gRPC layer. If the intention of this change is showing the connection type and subtype, storing these attributes are redundant because we're already have such information in the `connections` table.

You just need to fetch the connection from the database and expose it.